api/swagger.yaml

```diff
@@ -6170,6 +6170,8 @@ definitions:
       Expected:
         description: |
           Commit ID of external tool expected by dockerd as set at build time.
+
+          **Deprecated**: This field is deprecated and will be omitted in a API v1.49.
```

Comment:
This is in a response, right?  So to _actually_ omit it in 1.49+ we'll have to do something like `omitempty` on it (or a split/separate struct)?  Would we then just make sure the code setting the value is conditional?  (Making sure we're planning for how that's going to work/happen, and maybe even how we'll remember to implement that since the implementation isn't included in this PR that I can see :sweat_smile:  perhaps it could be, since we do already know which API version it should be removed in?)

Reply:
Yes, this PR only did the deprecation part, but I had follow-up changes stashed to handle the `omitempty` and to make it conditional; that still required an API version bump, so didn't push those changes yet, and I kept the `omitempty` out, in case there would be contention around "omit" or "keep, but empty".


Just pushed a WIP branch with those stashed changes if you're interested in the approach I had in mind 😅

Reply:
Forgot to post the link;

- https://github.com/moby/moby/pull/48556